src/server/api/general.js

```diff
@@ -269,6 +269,50 @@ export default function useGeneralApi(app) {
         recordWebEvent(this, 'api/accounts', account ? account.name : 'n/a');
     });
 
+    router.post('/create_user', koaBody, function*() {
+        if (rateLimitReq(this, this.req)) return;
+
+        const { name, email, secret } =
+            typeof this.request.body === 'string'
+                ? JSON.parse(this.request.body)
+                : this.request.body;
+
+        logRequest('create_user', this, { name, email });
+
+        try {
+            if (secret !== process.env.CREATE_USER_SECRET)
+                throw new Error('invalid secret');
+
+            if (!emailRegex.test(email.toLowerCase()))
+                throw new Error('not valid email: ' + email);
+            const existingUser = yield findUser({
+                email: esc(email),
+                name: esc(name),
+            });
+            if (existingUser) {
+                this.body = JSON.stringify({
+                    success: false,
+                    error: 'user with this email or name already exists',
+                });
+                this.status = 400;
+            } else {
+                const user = yield models.User.create({
```

Comment:
This isn't enough to create an user that can be recovered. We need an `Account` instance created and attached to the `User` as well as an `Identity` instance with the email.

Reply:
ah, ok. i thought you just wanted the user record. for the Account row, what should we use for `owner_id`? and for Identiy what should we use for `provdier` and `provider_id`? these columns are referenced in the account recovery code.

Reply:
For Identity set it up as an verified email, not sure about owner_id check what the creation process is doing now and set it to the same. This endpoint should do the same thing as the last step of the current signup process minus the blockchain account creation which will have happened already when this endpoint is hit by faucet

Reply:
sorry i meant `owner_key` -- it's generated here: https://github.com/steemit/condenser/blob/master/src/app/components/pages/CreateAccount.jsx#L86-L114

Reply:
i pushed a change to create the `account` and `identity` rows -- no keys yet

Reply:
Ah, good question. We might need to pass the public key along from faucet as well...